### ISSUE TITLE
Randomize default guest account instance

### ISSUE THREAD

**Issue by Rexios80** [mlemgroup/mlem#1330]
## Introduction
Lemmy.world already has too large of a share of the Lemmy user-base

## Requirements
Randomize the instance the default guest account uses

## Discussion
Similar work has been done for the join-lemmy.org site
https://github.com/LemmyNet/joinlemmy-site/pull/358
https://github.com/LemmyNet/joinlemmy-site/pull/359

**Comment by Sjmarf** [comment]
Yeah, I do want to avoid recommending lemmy.world. When the user tries to create an account using Mlem, I think it's a good idea to recommend a variety of smaller general-purposes instances. We'd either choose these ourselves or pull the list from an external source (possibly awesome-lemmy-instances) if a good enough list exists already. I want to avoid giving a random selection from the entire list, because there are certain instances that are interest-based or widely defederated that users may regret joining later. 

When the user first downloads the app at the moment, we put them straight on lemmy.world (on the TestFlight). We'd rather put them straight on an instance rather than making them choose one so that they can start exploring without going through the hassle of choosing an instance. Randomizing the instances at this stage may also be a good idea, but again I think it should be from a preset list rather than completely random.
